CIScripts/InitializeCIScript.ps1

```diff
@@ -5,6 +5,8 @@ if ($Env:ENABLE_TRACE -eq $true) {
 
 # Refresh Path
 $Env:Path = [System.Environment]::GetEnvironmentVariable("Path", "Machine")
+$Env:PSModulePath
+$Env:Path
```

Comment:
Potrzebujemy tego wypisywania? brzydko wygląda

Reply:
feel free to add commits to this PR, let's treat it as a working branch